OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
+
+                    if (useDistortion)
                     {
                         //@todo figure out which type of distortion to use
                         //right now, there is only one option, SurfaceRadialDistortion
                         //get distortion parameters
-                        OSVR.ClientKit.RadialDistortionParameters distortionParameters =
-                        Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(
-                        Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
+                        OSVR.ClientKit.RadialDistortionParameters distortionParameters;
+
+                        try
+                        {
+                            distortionParameters = Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
```

Comment:
It always fail on my three PC with various HDKs yes. So the hardcoded values are here just in case. I know it's not the best way but it allow Linux and Mac users to have a better quality.